frontend/app/services/GuardianContentService.scala

```diff
@@ -1,21 +1,18 @@
 package services
 
-import java.time.Instant
-
 import akka.actor.ActorSystem
-import com.gu.contentapi.client.model.{ContentApiError, ItemQuery, SearchQuery}
-import com.gu.contentapi.client.model.v1._
 import com.gu.contentapi.client.GuardianContentClient
+import com.gu.contentapi.client.model.v1._
+import com.gu.contentapi.client.model.{ContentApiError, ItemQuery, SearchQuery}
 import com.gu.memsub.util.ScheduledTask
-import configuration.Config
-import org.joda.time.DateTime
 import com.gu.monitoring.SafeLogger
 import com.gu.monitoring.SafeLogger._
-import play.api.libs.iteratee.{Enumerator, Iteratee}
+import configuration.Config
```

Comment:
https://github.com/playframework/play-iteratees
is end of life and has no scala 2.13 release

I have rewritten the code in this file that uses it in order to use recursion instead.  This is a recursive flatMap so it's not tail recursive, but it should be ok for now.  I would be possible to make it tail recursive I'm sure, but I have't done so.